deploy/crds/capabilities.3scale.net_openapis_crd.yaml

```diff
@@ -0,0 +1,204 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: openapis.capabilities.3scale.net
+spec:
+  group: capabilities.3scale.net
+  names:
+    kind: Openapi
```

Comment:
I'd prefer naming our kind OpenAPI to match with the conventions with known accronyms and what we are doing with other kinds (`APIManager`, APIcast,...).

Reply:
If we change this keep in mind that the struct names (Go data types) should be changed too